### PR TITLE
Add RFC for Removing Custom App Namespace.

### PR DESCRIPTION
**Pull request by weerd**
This pull request introduces the **"Removing Custom App Namespace" RFC**:

> We should remove the custom application namespaces from the Laravel PHP projects that utilize them.

### CODE REVIEW

File: 012-remove-custom-app-namespace.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+We should remove the custom application namespaces from the Laravel PHP projects that utilize them.
+
+**Discussion:** <https://github.com/DoSomething/rfcs/pull/12>
+
+## Problem
+
+A few of our apps utilize custom application namespaces for classes, which adds to the level of complexity and thoroughness required when performing upgrades or running commands, etc.
+
+Prior to Laravel 6.x, there was an [`app:name`](https://github.com/laravel/framework/blob/5.8/src/Illuminate/Foundation/Console/AppNameCommand.php) Artisan command you could run after initially setting up a project to rename the application namespace from `App\\` to something custom (e.g. we use `Auora\\` and `Rogue\\` in those respective projects).
```
Comment (DFurnes) [inline 012-remove-custom-app-namespace.md]: Small typo:

```suggestion
Prior to Laravel 6.x, there was an [`app:name`](https://github.com/laravel/framework/blob/5.8/src/Illuminate/Foundation/Console/AppNameCommand.php) Artisan command you could run after initially setting up a project to rename the application namespace from `App\\` to something custom (e.g. we use `Aurora\\` and `Rogue\\` in those respective projects).
```

**Review by DFurnes — APPROVED** [review]
I'm in favor of this! :+1:

**Comment by DFurnes** [comment]
This is really sensible & something we're going to tackle as part of other upcoming simplification work!

Going to merge this in. ⛵

**Comment by mendelB** [comment]
Gah missed this one! post-merge 👍 this makes great sense!!

**Pull request merged**